website/content/en/docs/faqs/_index.md

```diff
@@ -4,6 +4,10 @@ linkTitle: FAQ
 weight: 10
 ---
 
+## What's the different between kubebuilder and operator-sdk?
+
+Currently, kubebuilder is used as a plugin for many of the operator-sdk features around golang-based operators. Thus, much of the scaffolding operator-sdk generates for a golang operator should look familiar if you are familiar with kubebuilder. See [this post] for further discussion of how kubebuilder is used within operator-sdk. Note that pre-1.0.0 versions of operator-sdk did not use kubebuilder, although they still used controller-runtime to generate scaffolding. See [this issue](https://github.com/operator-framework/operator-sdk/issues/1758) for past discussion of legacy differences.
+
```

Comment:
```suggestion
SDK and Kubebuilder are integrated projects which have the purpose to help you to build your operator projects. Both tools leverage on [controller-runtime][controller-runtime] and will follow the same standard layout. For further information see the doc [What’s in a basic project?][kb-doc-what-is-a-basic-project]. 

However, SDK provides features to help you with some actions such as; to integrate your project with [OLM][olm] and to distribute your projects on [OperatorHub.io][operatorhub.io] as the OCP catalogue, [Scorecard][scorecard-doc] to help you test your projects. Nonetheless, SDK also allows you to work with other languages such as Ansible and Helm to create operators as well. 

Also, to have a better context over the advantages to use SDK as its differences over Kubebuilder, check the [comment][issue-comment] and the blog post [Operator SDK Reaches v1.0][operator-sdk-reaches-v1.0]. 

```

Reply:
Following the links:


[controller-runtime]: https://github.com/kubernetes-sigs/controller-runtime
[kb-doc-what-is-a-basic-project]: https://book.kubebuilder.io/cronjob-tutorial/basic-project.html
[olm]:  https://github.com/operator-framework/operator-lifecycle-manager
[operatorhub.io]: https://operatorhub.io/
[scorecard-doc]: https://sdk.operatorframework.io/docs/advanced-topics/scorecard/
[operator-sdk-reaches-v1.0]: